mappers/gige/driver/gigeclient.go

```diff
@@ -293,6 +293,7 @@ func (gigEClient *GigEVisionDevice) PostImage(DeviceSN string) {
 	go func() {
 		var buffer []byte
 		var bufferHdr = (*reflect.SliceHeader)(unsafe.Pointer(&buffer))
+		defer C.free_image((**C.char)(unsafe.Pointer(&imageBuffer)))
```

Comment:
This func can free the memory after the func ends and avoid the problem of memory leak